src/actions/__tests__/contactsActions.test.js

```diff
@@ -186,7 +194,7 @@ describe('Contacts Actions', () => {
       });
     });
 
-    it('should return and do nothing if accessToken does not exist', async () => {
+    xit('should return and do nothing if accessToken does not exist', async () => {
```

Comment:
```suggestion
    it('should return and do nothing if accessToken does not exist', async () => {
```

Reply:
This test needs updating since accesskeys will be removed (or invalid in new_connections)